hail/python/hail/context.py

```diff
@@ -352,3 +352,16 @@ def upload_log():
     """
 
     Env.hc()._jhc.uploadLog()
+
+
+def _set_flags(**flags):
```

Comment:
Who calls this function?

Reply:
I do. :P

Reply:
Less facetiously---with this PR, toggling
```
hl.read_matrix_table(file)._force_count_rows()
hl._set_flags(cpp="1")
hl.read_matrix_table(file)._force_count_rows()
hl._set_flags(cpp=None)
hl.read_matrix_table(file)._force_count_rows()
```
the first read will be done with the JVM decoder, and the second will be done with the C++ one. Eventually, when we get enough stability into the c++ stuff for e.g. @konradjk to play with, this will be a more convenient way of being able to toggle that.